events.js

```diff
@@ -1,13 +1,89 @@
+    listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveUrl: 'https://alexrs90.github.io/Portfolio/',
+    githubUrl: 'https://www.github.com/AlexRS90/Portfolio',
+  },
+  {
+    title: 'Multi-Post Stories',
+    img: 'images/yogaDesktop.png',
+    description: 'Up maids me an ample stood given. Certainty say suffering his him collected intention promotion. Hill sold ham men made lose case. Views abode law heard jokes too. Was are delightful solicitude discovered collecting man day. Resolving neglected sir tolerably but existence conveying for. Day his put off unaffected literature partiality inhabiting.',
+    listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveUrl: 'https://alexrs90.github.io/Portfolio/',
+    githubUrl: 'https://www.github.com/AlexRS90/Portfolio',
+  },
+  {
+    title: 'School Web Page',
+    img: 'images/thumpnailTemplate.png',
+    description: 'Arrival entered an if drawing request. How daughters not promotion few knowledge contented. Yet winter law behind number stairs garret excuse. Minuter we natural conduct gravity if pointed oh no. Am immediate unwilling of attempted admitting disposing it. Handsome opinions on am at it ladyship. ',
+    listOfTechnologies: ['HTML', 'Bootstrap', 'Ruby'],
+    liveUrl: 'https://alexrs90.github.io/Portfolio/',
+    githubUrl: 'https://www.github.com/AlexRS90/Portfolio',
+  },
+];
+
+for (let i = 0; i <= 1; i += 1) {
+  let cards = '';
+  let index = 1;
+  Object.keys(projects).forEach((project) => {
+    cards += `<div class="gridItem grid3 backGroundImg"><h2>${projects[project].title}</h2><p>A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups required. has been the<br> industry's standard</p><ul class="d-flex"> <li><a href="#"> <img src="images/htmlTransparent.png" alt="htmlButton" class="buttonPropertiesWorks"></a></li><li><a href="#"> <img src="images/bootstrapTransparent.png" alt="bootstrapButton" class="buttonPropertiesWorks"></a></li><li><a href="#"><img src="images/rubyTransparent.png" alt="rubyButton" class="buttonPropertiesWorks"></a> </li> </ul><input class="btnPrimary buttonProperties project-${index}" type="button" value="See Project"></div>`;
+    index += 1;
   });
+  document.querySelector('#works-section').innerHTML += cards;
+  cards = '';
+  index = 1;
+}
+
+const mobileBurger = document.querySelectorAll('.burger');
+const mobileMenu = document.querySelector('#mobile-menu');
+const menuItem = document.querySelectorAll('.menu-item');
+const popupToggle = document.querySelectorAll('.buttonProperties');
+const popupWindow = document.querySelector('.pop-up-main');
+const popupCloser = document.querySelector('.close-pop-up');
+const popupTitle = document.querySelector('.pop-up-title');
+const popupText = document.querySelector('.pop-up-text');
+const popupOpener = document.querySelector('.buttonOpener');
 
-function check (){
+function check() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.